robocop/checkers/misc.py

```diff
@@ -266,7 +266,7 @@
             name="max_width",
             default=80,
             converter=int,
-            desc="Maximum width of IF below which it will be recommended to use inline IF",
+            desc="Maximum width of IF (in character) below which it will be recommended to use inline IF",
```

Comment:
I missed the 's' in the previous suggestion:
```suggestion
            desc="Maximum width of IF (in characters) below which it will be recommended to use inline IF",
```